src/components/ListingDetails/index.tsx

```diff
@@ -0,0 +1,320 @@
+    getValue: (data: Listing) => (data as Interpretation).languages.join(', '),
+  },
+  // Interpretation Type
+  {
+    label: 'Interpretation Type', // 'Language Support Type' for consistency w/ Doc?
+    getValue: (data: Listing) => 'One-Time Intepretation',
+  },
+  // Remote/In Person
+  {
+    label: 'Remote/In Person',
+    getValue: (data: Listing) =>
+      (data as Interpretation).is_remote ? 'Remote' : 'In Person',
+  },
+];
+
+const docFields = [
+  // Languages
+  {
+    label: 'Language(s)',
+    getValue: (data: Listing) =>
+      (data as DocumentTranslation).languages.join(', '),
+  },
+  // Language Support Type
+  {
+    label: 'Language Support Type',
+    getValue: (data: Listing) => 'Written',
+  },
+  // Number of Pages
+  {
+    label: 'Number of Pages',
+    getValue: (data: Listing) => (data as DocumentTranslation).num_pages,
+  },
+];
+
+const LCAFields = [
+  // Country Field
+  {
+    label: 'Country Field',
+    getValue: (data: Listing) => (data as LimitedCaseAssignment).country,
+  },
+  // Language(s)
+  {
+    label: 'Language(s)',
+    getValue: (data: Listing) =>
+      (data as LimitedCaseAssignment).languages.join(', '),
+  },
+  // Expected Deliverable
+  {
+    label: 'Expected Deliverable',
+    getValue: (data: Listing) => (data as LimitedCaseAssignment).deliverable,
+  },
+];
+
+export default function ListingDetails({
+  listingData,
+}: {
+  listingData: Listing;
+}) {
+  const listingFields = () => {
+    if (listingData.listing_type === 'CASE') {
+      if (listingData.needs_interpreter) {
+        return caseInterpretationFields;
+      }
+      // will we ever need to render the case listing (instead of case interp) for a case that has needs_interpreter === true?
+      return caseFields;
+    }
+    if (listingData.listing_type === 'INT') {
+      return intepretationFields;
+    }
+    if (listingData.listing_type === 'DOC') {
+      return docFields;
+    }
+    return LCAFields;
+  }
+
+  const dateComponent = () => {
+    if (listingData.listing_type !== 'INT') {
+      let dateHeader = 'Assignment Deadline';
+      let dateData = '';
+      if (listingData.listing_type === 'CASE') {
+        dateHeader = 'Next Court/Filing Date';
+        dateData = formatTimestamp(listingData.upcoming_date); // listingData.upcoming_date ? formatTimestamp(listingData.upcoming_date) : "N/A"
+      } else {
+        dateData = formatTimestamp(listingData.deadline);
+      }
+      return (
+        <IconTextGroup>
+          <Icon type="calendar" />
+          <DateText $bold>
+            {dateHeader}: <DateText> {dateData}</DateText>
+          </DateText>
+        </IconTextGroup>
+      );
+    } 
+  };
+
+  const profile = useContext(ProfileContext);
+
+  const Interest = useMemo(() => {
+    if (profile && profile.userId)
+      return profile.profileData ? (
+        // <InterestForm caseData={listingData} />
+        <H3>Updated Interest Form Coming Soon!</H3> // remove after refactoring InterestForm
```

Comment:
quick little nit: can we show the interest form for the cases page? To do so, we can update this line to: 
`<InterestForm caseData={listingData as CaseListing} />`

We'd also need to update line 317 the following code:
`      <BorderedSection>
        {listingData.listing_type === 'CASE' ? (
          Interest
        ) : (
          <H3>Updated Interest Form Coming Soon!</H3>
        )}
      </BorderedSection>`

Reply:
sg! added it back